scripts/post_build.js

```diff
@@ -3,70 +3,81 @@
  */
 const fs = require('fs');
 const path = require('path');
+const { execSync } = require('child_process');
 const packageJson = require('../package.json');
 const packageTemplate = require('./package.template.json');
+const moduleExports = import('../rollup.module-exports.mjs');
 
-const { version } = packageJson;
-const getDistPath = (subPath) => path.resolve(__dirname, subPath != null ? `../dist${subPath}` : '../dist');
-
-function appendVersionToTypeDefn() {
-  const indexDtsPath = path.resolve(getDistPath(), 'index.d.ts');
-  const indexDts = fs.readFileSync(indexDtsPath, 'utf8');
-  const indexDtsWithVersion = indexDts.replace(
-    'Type Definitions for @sendbird/uikit-react@{{ version }}',
-    `Type Definitions for @sendbird/uikit-react@${version}`,
-  );
-
-  fs.writeFileSync(indexDtsPath, indexDtsWithVersion);
-}
+const getDistPath = (subPath) =>
+  path.resolve(__dirname, subPath != null ? `../dist${subPath}` : '../dist');
 
 /**
  * Remove all the unnecessary fields from package.json in root of the project
  */
-function cleanupPackageJSON(json) {
-  /**
-   * We remove scripts, module, main, types, and private fields from package.json
-   * Becuase folder structure is different in dist
-   * And, we don't want to run obligatory scripts in dist
-  */
-  const cleanedPkgJSON = {
-    version: json.version,
+async function createPackageJSON(json) {
+  const exportList = (await moduleExports).default;
+
+  const typesVersions = { '*': {} };
+  const exports = {
+    './package.json': './package.json',
+    './dist/index.css': './dist/index.css',
```

Comment:
Since it's already being distributed that way, we need to maintain it (for backward compatibility).
If we update to v4, then we can move it to the top level at that time

Reply:
ah i see. Great, thanks!